vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -18,7 +18,7 @@ The following steps utilize the ent bundle CLI.
 ``` sh
 ent bundle svc start keycloak
 ```
-2. Start up the Spring Boot application `conference-ms`. The logs will display in the console and you can stop the application with `Ctrl+C`. Keep the microservice running while the MFEs run.
+2. Start up the Spring Boot application `conference-ms`. The logs will display in the console and you can stop the application with `Ctrl+C`. Keep the microservice running while the MFEs are run.
```

Comment:
my vote is "MFEs" are running (common phrasing/understanding trumps the word repetition) but if you're set against that then "MFEs run" like you had before

Reply:
the run here is used in the command sense of the word, as in run the command. The tutorial is about running the components. I think this is fine and more accurate.

Reply:
i can appreciate that, but i think it's either "while the MFEs are running" or "when the MFEs are run"... "while" implies continued duration, whereas executing a command is done in an instant